Docs: Healthcheck on gateway-proxy-ssl (#7413)
solo-io/gloo: docs/content/guides/traffic_management/request_processing/health_checks/_index.md

    options:
      healthCheck:
        path: /any-path-you-want
```
{{% /tab %}} 
{{% tab name="gateway-proxy-ssl" %}}
```yaml
apiVersion: gateway.solo.io/v1
kind: Gateway
metadata:
  labels:
    app: gloo
  name: gateway-proxy-ssl
  namespace: gloo-system
spec:
  bindAddress: '::'
  bindPort: 8443
  httpGateway:
    options:
      healthCheck:
        path: /any-path-you-want
  proxyNames:
  - gateway-proxy
  ssl: true
  useProxyProto: false
```
{{% /tab %}} 
   {{< /tabs >}}

> 3. To test the health check, drain the Envoy connections by sending an `HTTP POST` to the Envoy admin port on `<envoy-ip>:<admin-addr>/healthcheck/fail`. This port defaults to `19000`.

Reviewer: ```suggestion
3. To test the health check, drain the Envoy connections by sending an `HTTP POST` request to the Envoy admin port on `<envoy-ip>:<admin-addr>/healthcheck/fail`. This port defaults to `19000`.
```

> 3. To test the health check, force Envoy into a draining state by sending an `HTTP POST` to the Envoy admin port on `<envoy-ip>:<admin-addr>/healthcheck/fail`. This port defaults to `19000`.

Reviewer 2: nbd if not, but is there a copiable curl POST for this?

> 3. To test the health check, force Envoy into a draining state by sending an `HTTP POST` to the Envoy admin port on `<envoy-ip>:<admin-addr>/healthcheck/fail`. This port defaults to `19000`.

Reviewer 2: Is there a different way to phrase the draining state thing here and in the next step? maybe something like:
```suggestion
3. To test the health check, drain the Envoy connections by sending an `HTTP POST` to the Envoy admin port on `<envoy-ip>:<admin-addr>/healthcheck/fail`. This port defaults to `19000`.
```
